app/views/publishers/vacancies/job_applications.html.slim

```diff
@@ -0,0 +1,75 @@
+- content_for :page_title_prefix, current_organisation.name
+
+= render BannerComponent.new do
+  - if deadline_passed?(@vacancy.expires_at)
+    = govuk_breadcrumbs breadcrumbs: { "#{t("publishers.vacancies_component.expired.tab_heading")}": jobs_with_type_organisation_url(:expired), "#{@vacancy.job_title}": organisation_job_job_applications_url(@vacancy.id) }
+  - else
+    = govuk_breadcrumbs breadcrumbs: { "#{t("publishers.vacancies_component.published.tab_heading")}": organisation_url, "#{@vacancy.job_title}": organisation_job_job_applications_url(@vacancy.id) }
+  .govuk-grid-row
+    .govuk-grid-column-full
+      h2.govuk-caption-l = t("publishers.vacancies.job_applications.applicants.heading")
```

Comment:
I don't think you need the first `publishers.vacancies.job_applications` - they are implied by the file location.

Reply:
applies across several translations

Reply:
For my taste, "applicants" is unnecessary